balboa-http/src/main/scala/com/socrata/balboa/server/rest/MetricsRest.scala

```diff
@@ -32,10 +34,9 @@ object MetricsRest {
   val periodMeter = com.yammer.metrics.Metrics.newTimer(
     classOf[MetricsRest], "period queries", TimeUnit.MILLISECONDS, TimeUnit.SECONDS)
 
-  val ds = DataStoreFactory.get()
 
   def get(entityId: String, params: Params, accepts: Seq[String]): ResponseWithType = {
-    val period = params.get(PeriodKey).map(Extractable[Period].extract) match {
+    val period = params.get("period").map(Extractable[Period].extract) match {
```

Comment:
Didn't you just introduce the `PeriodKey` constant in a recent commit? Are you taking it out here?